pkg/controllers/management/node/utils.go

```diff
@@ -58,8 +58,8 @@ func buildCreateCommand(node *v3.Node, configMap map[string]interface{}) []strin
 	for k, v := range configMap {
 		dmField := "--" + sDriver + "-" + strings.ToLower(regExHyphen.ReplaceAllString(k, "${1}-${2}"))
 		switch v.(type) {
-		case int64:
-			cmd = append(cmd, dmField, strconv.FormatInt(v.(int64), 10))
+		case float64:
+			cmd = append(cmd, dmField, strconv.FormatInt(int64(v.(float64)), 10))
```

Comment:
You are assuming all the float64 type as in64. Please do a type check to tell int64 from float 64.

Reply:
Updated!

Reply:
You are still doing `int64(v.(float64))`. You are parsing a float value to int without any check.

Please do `fmt.Sprintf("%v",v.(float64))`